addons/base_import/models/base_import.py

```diff
@@ -235,25 +238,29 @@ def _read_xls_book(self, book):
                     values.append(cell.value)
             if any(x for x in values if x.strip()):
                 yield values
+
+    # use the same method for xlsx and xls files
     _read_xlsx = _read_xls
 
-    def _read_ods(self, record, options):
-        doc = odf_ods_reader.ODSReader(file=io.BytesIO(record.file))
+    @api.multi
+    def _read_ods(self, options):
+        """ Read file content using ODSReader custom lib """
+        doc = odf_ods_reader.ODSReader(file=io.BytesIO(self.file))
 
         return (
             row
             for row in doc.getFirstSheet()
             if any(x for x in row if x.strip())
         )
 
-    def _read_csv(self, record, options):
+    @api.multi
+    def _read_csv(self, options):
         """ Returns a CSV-parsed iterator of all empty lines in the file
-
-        :throws csv.Error: if an error is detected during CSV parsing
-        :throws UnicodeDecodeError: if ``options.encoding`` is incorrect
+            :throws csv.Error: if an error is detected during CSV parsing
+            :throws UnicodeDecodeError: if ``options.encoding`` is incorrect
```

Comment:
All the indentation changes in docstrings should probably be avoided, they blow up the diff for no reason.